server/db/migrations/25_service_deploy_queued_at.rb

```diff
@@ -0,0 +1,6 @@
+class ServiceDeployQueuedAt < Mongodb::Migration
+  def self.up
+    GridServiceDeploy.create_indexes
+    GridServiceDeploy.all.set(queued_at: Time.now.utc)
```

Comment:
This could also be something like `1970-01-01 12:00:00` or whatever.... I don't think it makes any difference, unless there is some `GridServiceDeploy` without any `started_at` set hanging around... those should always be very recent deploys, and it's okay to just run those?